CHANGELOG.md

```diff
@@ -1,10 +1,23 @@
 # Changelog
 All notable changes to this project will be documented in this file - [read more](docs/changelog.md).
 
-## [Unreleased]
+## [UNRELEASED]
+
+## [0.10.0]
+
+**WARNING: THIS IS  A BREAKING CHANGE RELEASE**
```

Comment:
Would it be possible to add short description who will this breakage affect ? 
I think there is a non insignificant subset of users who will not be affected, lets not make them worry too much ?

Unless I'm mistaken ofcourse and everyone will have to perform some migration ?

Reply:
Everyone should perform some upgrade. but I will add.

Reply:
good idea actually